pkg/sources/git/git_test.go

```diff
@@ -315,3 +316,44 @@ func TestSource_Chunks_Edge_Cases(t *testing.T) {
 		})
 	}
 }
+
+func TestPrepareRepo(t *testing.T) {
+	tests := []struct {
+		uri    string
+		path   string
+		remote bool
+		err    error
+	}{
+		{
+			uri:    "https://github.com/dustin-decker/secretsandstuff.git",
+			path:   "/var/folders/1m/m4kl5q2102bfr4k4566nmf800000gn/T/trufflehog2276311773",
```

Comment:
See the other MR comments, we shouldn't expect a particular path here because it's OS dependent.

Reply:
great point.

Reply:
worth noting this can probably made a bit better with some more thought, but this should also work for the time being as the concept is tested I think well enough.